scripts/hack-express-server.js

```diff
@@ -0,0 +1,21 @@
+const commander = require('commander');
```

Comment:
What is hack-express-server needed for?

Reply:
Won't teams need access to this in terra-functional-testing or terra-webpack to run their site in prod mode?

Reply:
heroku deployment see: https://github.com/cerner/terra-toolkit/pull/444#discussion_r495950265

Reply:
Can we at least name this something other than 'hack' 😋  and then maybe an inline comment with an issue reference so this work doesn't get forgotten?

Reply:
I'd like to keep the blatant hack name until we fix it for real. (which should be soon with the wdio 6 updates if i get my way.)